draft-ietf-rats-architecture.md

```diff
@@ -69,27 +69,113 @@ remote attestation procedures (RATS).
 
 # Terminology {#terminology}
 
-    <this section can include content from Terminology in draft-birkholz-rats-architecture
-    and draft-thaler-rats-architecture>
+This document uses the following terms:
+
+* Appraisal Policy: A set of rules that direct how a system evaluates
+    the validity of information about another entity.  Compare /security policy/
+    in {{?RFC4949}}.
+
+* Attestation: A process by which one entity (the "Attester") provides evidence
```

Comment:
this term gets re-introduced here, without any annotation.
The appraisal policy are the rules on how to test the evidence. 
Here, "Attestation" is the creation of the evidence
Often people use the term "Attestation" to refer to the whole process.
Suggestion that we should define this term *somewhere*, but maybe "Remote Attestation"?

Reply:
FIDO has NONE-Attestation-Type, where there is no assessment.

Reply:
Let's take this to the WG, and maybe we can not get consensus on this term.

Reply:
Charter says: "Remote
attestation procedures (RATS) enable relying parties to establish a level of
confidence in the trustworthiness of remote system components through the
creation of attestation evidence by remote system components and a processing
chain towards the relying party."